base_app/src/app/dataset-tables/title-bar-test.component.spec.ts

```diff
@@ -173,33 +178,14 @@ describe('TitleBarTest', () => {
         }
     });
 
-    it('should have created the AppComponent', () => {
-        expect(component).toBeDefined();
-    });
 
-    /*
-    it('should call getProps', (done) => {
-        expect(component).toBeDefined();
 
-        component.getProps();
 
-        waitUntil(function () {
-            return (component._propsSubscriptionState !== SubscriptionState.AwaitingAsyncResponse);
-        }).then(function () {
-            done();
-        });
+    it('should have created the AppComponent', () => {
+        expect(component).toBeDefined();
     });
 
-    it ('should receive async result from getProps()', (done) => {
```

Comment:
Would like to have a test that validates that props can / have been received.

Reply:
Never mind - future.

Reply:
I agree, definitely something good to test. I've moved it from the title-bar tests to the [AppComponent tests](https://github.com/greg-ses/simpleui/blob/35690d8f7860e0c52bf0d811246227caf9818587/base_app/src/app/app.component.spec.ts#L84) since I thought it made more sense to have it there. It's disabled at the moment until I can work through the service issue